index.js

```diff
@@ -6,6 +6,8 @@ const fs = require('fs');
 let assert = require('assert');
 const mkdirp = require('mkdirp');
 const getDirName = require('path').dirname;
+const Container = require('../container');
+const Codeceptjs = require('../index');
```

Comment:
why you need this?

Reply:
Because we need container module for allure report. Updated my code again please review

Reply:
@puneet0191 updated it now please see